src/plone/rest/traverse.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from Products.CMFPlone.interfaces.siteroot import IPloneSiteRoot
+from ZPublisher.BaseRequest import DefaultPublishTraverse
+from plone.rest.interfaces import IAPIRequest
+from zope.component import adapts
+from zope.component import queryMultiAdapter
+from zope.interface import implements
+from zope.publisher.interfaces.browser import IBrowserPublisher
+
+NAME_PREFIX = u'rest_'
+
+
+class RESTTraverse(DefaultPublishTraverse):
+    adapts(IPloneSiteRoot, IAPIRequest)
+
+    def publishTraverse(self, request, name):
+        try:
+            obj = super(RESTTraverse, self).publishTraverse(request, name)
+        except KeyError:
```

Comment:
I noticed that here only `KeyError` is being handled, whereas in the wrapper below it's `except (KeyError, AttributeError)`. Is there a particular reason for this difference?